DataFormats/Detectors/ITSMFT/ITS/include/DataFormatsITS/TrackITS.h

```diff
@@ -88,25 +88,26 @@ class TrackITS : public o2::track::TrackParCov
   o2::track::TrackParCov& getParamOut() { return mParamOut; }
   const o2::track::TrackParCov& getParamOut() const { return mParamOut; }
 
-  void setPattern(uint16_t p) { mPattern = p; }
+  void setPattern(uint32_t p) { mPattern = p; }
   int getPattern() const { return mPattern; }
   bool hasHitOnLayer(int i) { return mPattern & (0x1 << i); }
+  bool isFakeOnLayer(int i) { return !(mPattern & (0x1 << (16 + i))); }
 
  private:
   o2::track::TrackParCov mParamOut; ///< parameter at largest radius
   ClusRefs mClusRef;                ///< references on clusters
   float mChi2 = 0.;                 ///< Chi2 for this track
-  uint16_t mPattern = 0;            ///< layers pattern
+  uint32_t mPattern = 0;            ///< layers pattern
 
-  ClassDefNV(TrackITS, 4);
+  ClassDefNV(TrackITS, 5);
```

Comment:
Hi @shahor02 , I am not sure of this, I bumped the class version because of the new size of `mBuffer` even though if I understood correctly its size was padded already to 32bits.